src/main/resources/static/style.css

```diff
@@ -1,31 +1,265 @@
+/* BASIC */
+
+html {
+  background-color: #56baed;
```

Comment:
dodaj proszę komentarze przy tych haszach jaki to kolor, latwiej się to czyta :)

Reply:
Dodane nazwy lub zbliżone odcienie  użytych kolorów.  Przy okazji usunąłem kilka lini css, które nie były użyte w kodzie.